packages/driver/src/cy/multidomain/index.ts

```diff
@@ -1,4 +1,6 @@
 import Bluebird from 'bluebird'
+import * as $Log from '../../cypress/log'
```

Comment:
bc of new esm changes this should be
```suggestion
import $Log from '../../cypress/log'
```

causes an error at runtime in those tests that are skipped with this PR, is there any test that would catch this?
![image](https://user-images.githubusercontent.com/14625260/132745428-6b3895e6-a521-4087-86f2-7aee0609868c.png)